apps/balancer-tools/src/contexts/PoolSimulatorContext.tsx

```diff
@@ -118,24 +119,25 @@ export function PoolSimulatorProvider({ children }: PropsWithChildren) {
     if (token) setCurrentTabToken(token);
   }
 
-  useEffect(() => {
-    if (initialData.poolParams === undefined) return;
-    if (
-      !initialData.poolType &&
-      !initialData.poolParams?.swapFee // all pool type have swapFee
-    )
-      return;
-    setInitialAMM(convertAnalysisDataToAMM(initialData));
-  }, [initialData]);
-
-  useEffect(() => {
-    if (
-      !customData.poolType &&
-      !customData.poolParams?.swapFee // all pool type have swapFee
-    )
-      return;
-    setCustomAMM(convertAnalysisDataToAMM(customData));
-  }, [customData]);
+  // TODO: BAL-539
+  // useEffect(() => {
+  //   if (initialData.poolParams === undefined) return;
+  //   if (
+  //     !initialData.poolType &&
+  //     !initialData.poolParams?.swapFee // all pool type have swapFee
+  //   )
+  //     return;
+  //   setInitialAMM(convertAnalysisDataToAMM(initialData));
+  // }, [initialData]);
+
+  // useEffect(() => {
+  //   if (
+  //     !customData.poolType &&
+  //     !customData.poolParams?.swapFee // all pool type have swapFee
+  //   )
+  //     return;
+  //   setCustomAMM(convertAnalysisDataToAMM(customData));
+  // }, [customData]);
```

Comment:
IMO we should delete the code we are not using at the moment because we still have this as a reference either on stableswap or in our git commit history. WDYT?